src/CabanaPD_Solver.hpp

```diff
@@ -327,13 +330,14 @@ class SolverElastic
     bool print;
 };
 
-template <class DeviceType, class ParticleType, class ForceModel,
-          class BoundaryCondition, class PrenotchType>
+template <class DeviceType, class InputType, class ParticleType,
```

Comment:
@streeve : What is ParticleType?

Reply:
`Particles`, but there are a number of different variants so it's easier to just template